backend/src/litigation_api/ingestion/pipeline.py

```
@@ -0,0 +1,121 @@
+                        domain_doc.ingestion_status = "FAILED_UPLOAD"
+                else:
+                    logger.info(f"Skipping download for unsupported document type or missing file URL: {domain_doc.doc_id} ({domain_doc.document_type})")
+                    domain_doc.ingestion_status = "SKIPPED_UNSUPPORTED_TYPE" # Adhere to unsupported document fallback behavior
+
+                domain_documents.append(domain_doc)
+
+            return domain_case, domain_dockets, domain_documents
+
+        if self.client:
+            return await _do_work(self.client)
+        else:
```

Ruff (0.15.11)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)